### ISSUE TITLE
Playframework 2.7 support

### ISSUE THREAD

**Issue by raboof** [gradle/gradle#6362]
Gradle hard-codes the Play versions it supports (https://github.com/gradle/gradle/blob/master/subprojects/platform-play/src/main/java/org/gradle/play/internal/platform/PlayMajorVersion.java#L30). Play 2.7.0 is in the works (2.7.0-M2 has been released), so it might be nice to update Gradle to support this series as well

### Expected Behavior

Being able to use the play plugin with Play 2.7.0-M2

### Current Behavior

Error message:

```
Exception thrown while executing model rule: PlayApplicationPlugin.Rules#createBinaries(ModelMap<PlayApplicationBinarySpec>, PlayApplicationSpecInternal, PlatformResolvers, PlayToolChainInternal, PlayPluginConfigurations, File, ProjectIdentifier) > create(binary)
   > Not a supported Play version: 2.7.0-M2. This plugin is compatible with: [PLAY_2_2_X, PLAY_2_3_X, PLAY_2_4_X, PLAY_2_5_X, PLAY_2_6_X].
```

**Comment by marcphilipp** [comment]
@raboof Is there a release date for the GA version of Play 2.7?

**Comment by raboof** [comment]
I think I heard mentions of September (but no huge changes are expected compared to the already-available 2.7.0-M2)

**Comment by marcphilipp** [comment]
@raboof Is there a publicly available timeline for 2.7?

**Comment by fellz** [comment]
Still no support for Playframework 2.7.0. Downloaded latest scr code 5.2.0-RC1 and still the same error.
Any plans to include 2.7.0 ?

**Comment by raboof** [comment]
Play 2.7.0 was literally released yesterday. Indeed gradle does not yet support it (otherwise this ticket would have been closed ;) ), but work on that is under way in #6374.

**Comment by orlovskyjavaprofi** [comment]
So when the new play will be supported, any ETA on this issue?

**Comment by lacasseio** [comment]
Play 2.7 support should be contributed to [gradle/playframework](https://github.com/gradle/playframework). Issue https://github.com/gradle/playframework/issues/92 is open to track this work.

I will leave this issue open until we finalize the last detail of the transition to the external plugin.

**Comment by arjunchhabra** [comment]
@lacasseio - Any update on this item? Whether extracting or merging here, I believe many of us are currently without a path forward.

There is a PR to enable support, here: https://github.com/gradle/gradle/pull/6374.

**Comment by arjunchhabra** [comment]
@lacasseio - Thanks, this makes sense and I understand that there are many priorities when aiming for a release. I will work on migrating that PR to the gradle/playframework plugin and will tag you once it is complete.

**Comment by arjunchhabra** [comment]
Hey @lacasseio, I've opened a PR with the changes here: https://github.com/gradle/playframework/pull/110. Please take a look at your convenience and let me know if there are any other items I should address. Thanks!

**Comment by lacasseio** [comment]
The PR was merged, I'm closing this.

**Comment by arjunchhabra** [comment]
Hey @lacasseio / @big-guy , the PR was merged into gradle/playframework but there are still failing tests and a release cannot be cut until the tests have been resolved, so there is still no release with 2.7 support. We are actively working on resolving the test failures here, but are new to the project and not sure how quickly we'll be able to turn around a solution. 

In the meanwhile, we have been blocked on this item for some time - wondering if it is possible to merge the previous PR (https://github.com/gradle/gradle/pull/6374) into the gradle repo directly as it was determined to work, as a temporary solution?
